### ISSUE TITLE
UndefVarError: SegyToSeis not defined

### ISSUE THREAD

**Issue by LeeyMessi** [SeismicJulia/Seismic.jl#58]
julia>using PyPlot,Seismic
julia>SegyToSeis("616_79_PR.SGY","616_79_PR.seis"):
ERROR:UnderVarError: SegyToSeis not defined
Please tell me what happened? thanks

**Comment by msacchi** [comment]
Hi 

it is working for me. 

using PyPlot,Seismic
download("http://seismic.physics.ualberta.ca/data/616_79_PR.SGY", "616_79_PR.SGY")
SegyToSeis("616_79_PR.SGY","616_79_PR.seis")

cheers, Mauricio

**Comment by LeeyMessi** [comment]
That's all right! I have solved this problem, because I had changed path of python, Thank you for your attention.
